### PR TITLE
feat: snackbar added

### PR DESCRIPTION
**Pull request by sumitch29**
### Snackbar added

-  Baseline snackbar
- Leading snackbar
- stacked snackbar

Screenshot (405)
Screenshot (406)
Screenshot (407)

### CODE REVIEW

File: Components/Toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
+/* ///////baseline///// */
+
+.baseline {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  font-size: large;
```
Comment (ShoaibTamboli) [inline Components/Toast/toast.css]: declare the value

File: Components/Toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  background-color: rgb(54, 53, 53);
+  width: fit-content;
+  border-radius: 5px;
+  padding: 5px 10px;
+}
+
+.baseline-para {
+  color: whitesmoke;
```
Comment (ShoaibTamboli) [inline Components/Toast/toast.css]: use css variable

Comment (sumitch29) [reply Components/Toast/toast.css]: ok. will change it.

File: Components/Toast/toast.html [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+                    <a class="sidebarlink" href="/Components/Rating/rating.html">Rating</a>
+                </li>
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/Grid/grid.html">Grid</a>
+                </li>
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/modal/modal.html">Modal</a>
+                </li>
+                <li class="sidebar-link active">
+                    <a class="sidebarlink" href="/Components/Toast/toast.html">Snackbar</a>
```
Comment (ShoaibTamboli) [inline Components/Toast/toast.html]: name files as per component. (easy to remember)

**Review by ShoaibTamboli — COMMENTED** [review]
looks good, just added some minor suggestions

File: Components/Toast/toast.html (hunk 3 above)
Comment (sumitch29) [reply Components/Toast/toast.html]: Thanks for pointing out. I will change it.

**Review by MonikaJhingan — APPROVED** [review]
The  code looks hood to be merged just use rem consistently in later projects.

File: Components/Toast/toast.css [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+}
+
+.baseline-retry {
+  display: flex;
+  color: rgb(223, 93, 223);
+  align-items: center;
+  padding: 10px 10px 10px 10px;
```
Comment (vidhi-thakur) [inline Components/Toast/toast.css]: You can use shorthand property for padding here.

File: Components/Toast/toast.css [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+}
+
+.baseline-cross {
+  color: whitesmoke;
+  display: flex;
+  align-items: center;
+  padding: 10px 15px 10px 15px;
```
Comment (vidhi-thakur) [inline Components/Toast/toast.css]: Same here as well, please use shorthand property

**Review by vidhi-thakur — APPROVED** [review]
Try and use shorthand property and css variables wherever possible.

File: Components/Toast/toast.css (hunk 4 above)
Comment (sumitch29) [reply Components/Toast/toast.css]: will change it

File: Components/Snackbar/snackbar.css [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+.stacked-cross:hover {
+  cursor: pointer;
+  background-color: var(--baseline-hover);
+  border-radius: 50%;
+}
+
+.stacked-label:hover {
+  cursor: pointer;
+}
```
Comment (vikrantkode) [inline Components/Snackbar/snackbar.css:139]: You can use all the snackbar hover classes at one place and style them all together 
eg:
.stacked-label : hover,
.leading-retry:hover{
cursor : pointer}

Same can be done for other also

**Review by vikrantkode — COMMENTED** [review]
Suggest some changes. look for it
Rest looks good to me.

**Pull request merged**